api/controllers/registerUser.py

```diff
@@ -51,10 +51,12 @@ def register_user():
 
         newUser.save_to_db()
 
-        perm = Permissions(isUser=True, user_permissions=newUser)
         if newUser.email in admins:
```

Comment:
How it is different from the latter one?

Reply:
the latter one did not seem to work either for me or fopr @ParthS007 .
So we discussed about it and I send this fix that works for everyone.